hydra-node/src/Hydra/Options.hs

```diff
@@ -361,6 +353,24 @@ data DirectChainConfig = DirectChainConfig
   deriving stock (Eq, Show, Generic)
   deriving anyclass (ToJSON, FromJSON)
 
+data BlockfrostChainConfig = BlockfrostChainConfig
+  { bfProjectPath :: FilePath
+  -- ^ Path to the blockfrost project file
+  , bfHydraScriptsTxId :: [TxId]
+  -- ^ Identifier of transaction holding the hydra scripts to use.
+  , bfCardanoSigningKey :: FilePath
+  -- ^ Path to the cardano signing key of the internal wallet.
+  , bfCardanoVerificationKeys :: [FilePath]
+  -- ^ Paths to other node's verification keys.
+  , bfStartChainFrom :: Maybe ChainPoint
+  -- ^ Point at which to start following the chain.
+  , bfContestationPeriod :: ContestationPeriod
+  , bfDepositDeadline :: DepositDeadline
+  -- ^ Deadline to detect deposit tx on-chain.
```

Comment:
Should: not duplicate these entries, but rather share them across implementations of `Direct` and `Blockfrost`

Also, we do not prefix field names according to our styleguide